mon-pix/tests/integration/components/routes/campaigns/assessment/skill-review-test.js

```diff
@@ -280,12 +274,10 @@ describe('Integration | Component | routes/campaigns/assessment/skill-review', f
       });
 
       context('when the organization has a customResultPageButtonUrl and a customResultPageButtonText', function() {
-        context('when the participant has finished a campaign with stages', function() {
+        context('when the participant has finished a campaign with stages and has a masteryPercentage and a idPixLabel', function() {
```

Comment:
**suggestion** : rajouter un test pour les cas : 
- `when the participant has finished a campaign with stages and has a masteryPercentage but no idPixLabel` ;
- `when the participant has finished a campaign with a masteryPercentage and idPixLabel`

Reply:
je pensais que vu qu'on testait ces cas dans les tests unitaires on pouvait s'en passer pour les tests d'intégration.

Reply:
oui c'est vrai